app/views/global_user_roles/edit.html.haml

```diff
@@ -0,0 +1,27 @@
+= content_for :h1 do
+  = t("pages.global_settings")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_global", sidenav_tab: "global_user_roles"
+
+%h2= t("pages.global_user_roles")
+
+= form_for(:user_role, url: global_user_role_path(@user.id), method: :put) do |form|
+  = form.error_messages
+  %ul.form
+    %li
+      = label_tag :username, nil, class: "require"
+      = text_field_tag :username, @user.username, disabled: true
+    %li
+      = label_tag :name, nil, class: "require"
+      = text_field_tag :name, @user.full_name, disabled: true
+    %li
+      = label_tag :email, nil, class: "require"
+      = text_field_tag :email, @user.email, disabled: true
+    %li
+      = form.label :roles, nil, class: "require"
+      = select_tag :roles, @user.global_role_select_options, multiple: true
+
+  %ul.inline
+    %li= form.submit t(".submit"), class: "btn"
```

Comment:
I would expect this to be disabled if submitting the form is going to generate an error because I can't edit myself.